internal/services/bot/bot_channel_directline_resource.go

```diff
@@ -133,6 +133,18 @@ func resourceBotChannelDirectline() *pluginsdk.Resource {
 					},
 				},
 			},
+
+			"extension_key1": {
+				Type:      pluginsdk.TypeString,
+				Computed:  true,
+				Sensitive: true,
+			},
+
+			"extension_key2": {
```

Comment:
as a style, we use `_1` for property names:
```suggestion
			"extension_key_1": {
				Type:      pluginsdk.TypeString,
				Computed:  true,
				Sensitive: true,
			},

			"extension_key_2": {
```